api/dist/apihelp/apihelp.html

```diff
@@ -207,6 +207,22 @@ <h2>Load data from a text file</h2>
                     </pre>
                 </div>
 
+                <div class="api">
+                    <h2>Search input</h2>
+                    <div id='map10search' style="width: 400px;"></div>
+                    <div id='map10' class="map"></div>
+                    <pre data-language="javascript">
+var map = new demo.Map({
+    div: 'map10',
+    zoom: 3,
+    center: [2544500, 1210100],
+    addSearch: true,
+    searchElementId: 'map10search'
```

Comment:
Do we relay needs the `addSearch`?
To be coherent with `div` shouldn't it be `searchDiv`?